Vagrantfile

```diff
@@ -1,12 +1,14 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 Vagrant.configure('2') do |config|
-  config.vm.box      = 'ubuntu/focal64' # 20.04
-  config.vm.hostname = 'rails-dev-box'
+  config.vm.box      = 'ubuntu/bionic64' # 18.04
```

Comment:
We could keep the most recent version, no need to change.

Reply:
SQL Server on Ubuntu doesn't support 20.04, see https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-ubuntu

Reply:
SQL Server on Ubuntu doesn't support 20.04, see https://docs.microsoft.com/en-us/sql/linux/quickstart-install-connect-ubuntu